toolbelt/generate-configs.js

```diff
@@ -3,29 +3,43 @@
  */
 
 const fs = require( 'fs' );
-const packageJsonTemplate = require( './templates/package.json' );
+const path = require( 'path' );
 
 const packageList = require('./packages-list.js').packageList();
 
 console.log('Generating package.json and tsconfig.json for:');
 
+const packagesRootDir = path.resolve(__dirname, '..', 'packages');
+const templatesRootDir = path.resolve(__dirname, 'templates');
+
+const packageJsonTemplate = require( path.resolve(templatesRootDir, 'package.json') );
+const tsconfigJsonTemplate = require( path.resolve(templatesRootDir, 'tsconfig.json') );
+
 for (let packageName of packageList) {
```

Comment:
This `for loop` is quite log.

It would be better to extract the content to a separate function and iterate through **forEach**.